src/main/java/server/Server.java

```diff
@@ -292,13 +275,13 @@ public void debugRun(JSONArray flags) {
     command.add("--debug=3");
 
     try {
-      System.out.println("Running from: " + HYPED_PATH);
+      System.out.println("Running from: " + buildDirectoryPath);
       terminalOutput = new JSONArray();
-      debugProcess = new ProcessBuilder(command).directory(new File(HYPED_PATH)).start();
+      debugProcess = new ProcessBuilder(command).directory(new File(String.valueOf(buildDirectoryPath))).start();
```

Comment:
@MiltFra changed it to use `Paths.get()` and ran it, seems to work the same but I think maybe now it's a relative path and not an absolute/global path. Can you please confirm what I've done is equivalent?